### PR TITLE
Refine InstructorStudentList/RecordsPage slice front-end side

### PR DESCRIPTION
**Pull request by wkurniawan07**
Refinement includes:
instructorStudentList.js
instructorStudentListAjax.js
instructorStudentRecords.js
instructorStudentList.jsp
instructorStudentRecords.jsp
And changes in HTML test files to adapt with the changing JSPs.

### CODE REVIEW

File: src/test/resources/pages/instructorStudentRecords.html [hunk 1]
```diff
@@ -121,16 +120,13 @@ <h4 class="modal-title">Benny Charlés's Profile - More Info</h4>
                                     </div>
                                     <div id="visibility-options0" class="panel panel-default" style="display: none;">
                                         <div class="panel-heading">Visibility Options</div>
-                                        <table class="table text-center" style="color:#000;">
+                                        <table class="table text-center" style="color: #000; background: #fff;">
```
Comment (damithc) [inline src/test/resources/pages/instructorStudentRecords.html]: Is this change intended?

Comment (wkurniawan07) [inline src/test/resources/pages/instructorStudentRecords.html]: The pre-refined jsp has this:
```
<table class="table text-center" style="color:#000;" style="background: #fff;">
```
which causes the second style to be ignored. I refined it to:
```
<table class="table text-center" style="color: #000; background: #fff;">
```
which allows both styles to take place.

Comment (damithc) [inline src/test/resources/pages/instructorStudentRecords.html]: I see. Try to separate behavioral changes into separate PRs, in case we
need to revert them later.

Comment (wkurniawan07) [inline src/test/resources/pages/instructorStudentRecords.html]: Alternatively, I can remove the second style altogether, since it's already ignored in the first place anyway. Is it better?

Comment (damithc) [inline src/test/resources/pages/instructorStudentRecords.html]: @kanghj what do you think?

Comment (kanghj) [inline src/test/resources/pages/instructorStudentRecords.html]: I think we can remove it for now. For now, it will not actually affect the actual background color of the table since the parent element of the table has a white background (due to bootstrap's panel class). The same table appears in the instructor's results page, with the same error, which I can fix in #3512

Comment (wkurniawan07) [inline src/test/resources/pages/instructorStudentRecords.html]: Ok then, reverting the extra background style before merging.
